app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -52,6 +52,29 @@ examples:
         text: "Use Government Gateway"
       - value: "govuk-verify"
         text: "Use GOV.UK Verify"
+  with_legend:
+    description: This adds a `legend` element containing the text
+    data:
+      name: "radio-group-legend"
+      heading: "What's it to do with?"
+      heading_level: 0
+      items:
+      - value: "yes"
+        text: "Yes"
+      - value: "no"
+        text: "No"
+  with_legend_custom_size:
```

Comment:
This feels like a duplication of the `with_custom_heading_size` example. I think it might be better to expand on the detail of the existing example, rather than add another.